question/bank/tagquestion/db/services.php

```diff
@@ -0,0 +1,39 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * External services definition for qbank_tagquestion.
+ *
+ * @package    qbank_tagquestion
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Safat Shahin <safatshahin@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$functions =
+    array(
+        'qbank_tagquestion_submit_tags_form' =>
+            array(
+                'classname' => 'qbank_tagquestion\external\qbank_tagquestion_external',
+                'methodname' => 'submit_tags_form',
+                'description' => 'Update the question tags.',
+                'type' => 'write',
+                'ajax' => true,
+            ),
+);
```

Comment:
In this line I'm getting this in codechecker: `#38: ); Line indented incorrectly; expected at least 4 spaces, found 0`